packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -372,9 +378,6 @@ export default class AgoraIpV4OrderController {
           .then((data) => {
             let countries = data;
```

Comment:
If at all we have countries coming from API how will we handling the organization filter ? 
We are dependent on region from service name if the country is coming as empty. And I see you have removed this logic. Shouldn't we need to keep as is and get the region based on countries ?